servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpContextKeys.java

```diff
@@ -33,6 +33,12 @@ public final class HttpContextKeys {
     public static final Key<HttpExecutionStrategy> HTTP_EXECUTION_STRATEGY_KEY =
             newKey("HTTP_EXECUTION_STRATEGY_KEY", HttpExecutionStrategy.class);
 
+    /**
+     * Tracks the original API mode used for client API conversions
+     */
+    public static final Key<HttpApiConversions.ClientAPI> HTTP_CLIENT_API_KEY =
```

Comment:
The `HttpContextKeys` is a holder for all public keys we expect users to use. If we ever add a special key for our internal purposes, we should not make it public.